caf-audit-service-container/pom.xml

```diff
@@ -297,86 +285,32 @@
                                 <env>
                                     <CAF_AUDIT_MODE>direct</CAF_AUDIT_MODE>
                                 </env>
-                                <ports>
-                                    <port>8080</port>
-                                </ports>
-                                <entryPoint>
-                                    <exec>
-                                        <args>/maven/tini-${tini.version}.exe</args>
-                                        <args>-v</args>
-                                        <args>--</args>
-                                    </exec>
-                                </entryPoint>
-                                <runCmds>
-                                    <runCmd>mv /opt/*.war $CATALINA_HOME/webapps</runCmd>
-                                </runCmds>
-                                <cmd>/start.sh</cmd>
+                                <healthCheck>
+                                    <mode>none</mode>
+                                </healthCheck>
                                 <assembly>
-                                    <basedir>/</basedir>
+                                    <basedir>/usr/share/tomcat</basedir>
                                     <mode>tar</mode>
                                     <inline>
-                                        <fileSets>
-                                            <fileSet>
-                                                <lineEnding>unix</lineEnding>
-                                                <includes>
-                                                    <include>*.sh</include>
-                                                </includes>
-                                                <fileMode>0755</fileMode>
-                                            </fileSet>
-                                        </fileSets>
                                         <dependencySets>
                                             <dependencySet>
-                                                <useProjectArtifact>false</useProjectArtifact>
+                                                <outputDirectory>webapps/caf-audit-service</outputDirectory>
```

Comment:
This was previously using the `${webserviceWarName}` token.

Reply:
Using the `${webserviceWarName}` token for this now.